flink-test-utils-parent/flink-connector-test-utils/src/main/java/org/apache/flink/connector/testutils/source/reader/SourceReaderTestBase.java

```diff
@@ -108,28 +103,31 @@ public void testAddSplitToExistingFetcher() throws Exception {
         }
     }
 
-    @Test(timeout = 30000L)
-    public void testPollingFromEmptyQueue() throws Exception {
+    @Test
+    @Timeout(30)
+    void testPollingFromEmptyQueue() throws Exception {
         ValidatingSourceOutput output = new ValidatingSourceOutput();
         List<SplitT> splits =
                 Collections.singletonList(getSplit(0, NUM_RECORDS_PER_SPLIT, Boundedness.BOUNDED));
         // Consumer all the records in the s;oit.
         try (SourceReader<Integer, SplitT> reader =
                 consumeRecords(splits, output, NUM_RECORDS_PER_SPLIT)) {
             // Now let the main thread poll again.
-            assertEquals(
-                    "The status should be ",
-                    InputStatus.NOTHING_AVAILABLE,
-                    reader.pollNext(output));
+            assertThat(reader.pollNext(output))
+                    .withFailMessage("The status should be %s", InputStatus.NOTHING_AVAILABLE)
+                    .isEqualTo(InputStatus.NOTHING_AVAILABLE);
         }
     }
 
-    @Test(timeout = 30000L)
-    public void testAvailableOnEmptyQueue() throws Exception {
+    @Test
+    @Timeout(30)
+    void testAvailableOnEmptyQueue() throws Exception {
         // Consumer all the records in the split.
         try (SourceReader<Integer, SplitT> reader = createReader()) {
             CompletableFuture<?> future = reader.isAvailable();
-            assertFalse("There should be no records ready for poll.", future.isDone());
+            assertThat(future.isDone())
+                    .withFailMessage("There should be no records ready for poll.")
```

Comment:
I think we should use`assertThat().as()` here instead of `withFailMessage()`. `withFailMessage()` will override the original generated "expected xxx but yyy" message, which is helpful for debugging. `as()` is designed for describing the assertion.